Frontend/src/components/Queue/QueueContainer.css

```diff
@@ -23,7 +23,6 @@ table {
 tbody {
   display: block;
   overflow: auto;
-  height: 1000px;
   width: 100%;
```

Comment:
I like not having as much space, and could we add a little bit more space at the top still? It looks a little squished now